README.md

```diff
@@ -37,6 +37,14 @@ You can also [donate some
 money](https://liberapay.com/IHateMoney/donate). All funds will be used
 to maintain the [hosted version](https://ihatemoney.org).
 
+Join the Contributers already contributing to this Repository.
```

Comment:
Typo: I believe it should be contribut**o**rs (no capital C, and o instead of e). Repository should also not be capitalized.

Something like "Join the other contributors" would work.